core/util_test.go

```diff
@@ -299,23 +300,21 @@ func (d *DrandTestScenario) RunDKG() *key.Group {
 	defer cancel()
 	done := make(chan bool) // signal we are done with the reshare before the timeout
 
-	runLeaderNode := func() {
+	runLeaderNode := func() error {
```

Comment:
If we make this change here, then the errors from `runLeaderNode` will not be passed as the function is launched with a `go runLeaderNode()`.

Reply:
updated to use the channel